recipe/meta.yaml

```diff
@@ -43,14 +44,13 @@ requirements:
     - llvm-spirv-{{ llvm_version.split(".")[0] }}
 
     - zlib                               # [osx]
-    - ld64 >=530                         # [osx]
+    - ld64 >=530,<711                     # [osx]
   run:
     - llvm-spirv-{{ llvm_version.split(".")[0] }}
     - ocl-icd                            # [linux]
     - khronos-opencl-icd-loader          # [osx]
-    - ld64 >=530                         # [osx]
+    - ld64 >=530,<711                     # [osx]
```

Comment:
Is there a way to say `!=711` rather than rule out any future versions?

Reply:
Yes, but future versions will probably have the same issue, and we would need another update to this package to also blacklist them... not sure what is better

Reply:
I'm OK with that. `ld64` updates seem few and far between anyway.

Reply:
Let's keep `<711`